_pages/about.md

```diff
@@ -50,9 +50,9 @@ journey started with studying Electrical Engineering at RWTH Aachen University
 in Germany. At the Institute for Communication Technologies and Embedded
 Systems, we worked as student assistants, where we contributed to different
 projects in the fields of hardware accelerators for machine learning, FPGA
-design, processor prototyping and embedded software. Nowadays, Bjoern works as a
-firmware developer in the audio industry. Kirthihan works as a digital hardware
-developer in the cybersecurity industry.
+design, processor prototyping and embedded software. Nowadays, Bjoern works as
+software engineer in the area of Electronic Design Automation (EDA). Kirthihan
```

Comment:
While we are already at it: could you change mine to "Kirthihan works as a software engineer in the field of System Simulation.

Reply:
ah right. Actually I wanted to ask you about it, but somehow I forgot :-D